packages/plugin-ext/src/plugin/types-impl.ts

```diff
@@ -1541,7 +1541,7 @@ export enum TaskScope {
 
 export class Task {
     private taskDefinition: theia.TaskDefinition | undefined;
-    private taskScope: theia.TaskScope.Global | theia.TaskScope.Workspace | theia.WorkspaceFolder | undefined;
+    private taskScope?: theia.TaskScope.Global | theia.TaskScope.Workspace | theia.WorkspaceFolder | undefined;
```

Comment:
why is the `?` needed, `undefined` is already supported.

Reply:
Removed